lib/galaxy/tools/evaluation.py

```diff
@@ -256,7 +256,7 @@ def wrap_input( input_values, input ):
         #       tools where the inputs don't even get passed through. These
         #       tools (e.g. UCSC) should really be handled in a special way.
         if self.tool.check_values:
-            param_dict[ "identifier_key" ] = dict((v, "%s|__identifier__" % k) for k, v in input_datasets.iteritems())  # allows lookup of identifier through HDA.
+            identifier_key_dict = dict((v, "%s|__identifier__" % k) for k, v in input_datasets.iteritems())  # allows lookup of identifier through HDA.
```

Comment:
@mvdbeek Please check!
It should be safe since `param_dict["identifier_key"]` has been introduced in #2029 and it is not used anywhere else.